tests/search/includes/classes/test-class-queue.php

```diff
@@ -1401,4 +1387,15 @@ protected static function get_method( $name ) {
 		$method->setAccessible( true );
 		return $method;
 	}
+
+	/**
+	 * @global wpdb $wpdb
+	 */
+	private function get_min_object_id_from_queue() {
+		/** @var wpdb $wpdb */
+		global $wpdb;
+
+		$table = $this->queue->schema->get_table_name();
+		return $wpdb->get_var( "SELECT MIN(job_id) FROM {$table}" );
+	}
```

Comment:
Because we have to use `DELETE` instead of `TRUNCATE`, the AUTO_INCREMENT value is no longer reset to 1. Therefore, we cannot rely upon the value of `job_id` anymore.